index.html

```diff
@@ -7,6 +7,6 @@
 	<tittle>Welcome to Potionfy</title>
 </head>
 <body>
-	<h1>Welcome to Potionfy! Enjoy your stay.</h1>
+	<h1>Welcome to Potionfy! Enjoy your stay!</h1>
```

Comment:
Could u remove the 'Enjoy ur stay' string and make the 'Welcome to Potionfy!' string red?